### PR TITLE
Update Rule “turn-emails-into-pbis/rule”

### PR DESCRIPTION
**Pull request by chrisschultzssw**
Automatically generated by Netlify CMS

### CODE REVIEW

File: rules/turn-emails-into-pbis/zendeskemail.png
Comment (kikibianc) [inline rules/turn-emails-into-pbis/zendeskemail.png:1]: Blur or delete this file - it has our internal email

**Review by kikibianc — CHANGES_REQUESTED** [review]
Blur or delete

File: rules/turn-emails-into-pbis/zendeskemail.png
Comment (chrisschultzssw) [reply rules/turn-emails-into-pbis/zendeskemail.png:1]: Done - I didn't end up using the image in the rule, so I have deleted it.

**Review by kikibianc — APPROVED** [review]
🦄🦄🦄

**Pull request merged**